### PR TITLE
Addressed comments from WGLC reviewers.

### PR DESCRIPTION
**Pull request by david-waltermire**
Addressed comments from Alex, Ira and Carsten.

### CODE REVIEW

File: draft-ietf-sacm-coswid.md [hunk 1]
```diff
@@ -257,17 +259,25 @@ In CBOR, an array is encoded using bytes that identify the array, and the array'
...
+All names registered with IANA according to requirements in section {{iana-value-registries}} also need to be valid according to the XML Schema NMToken data type (see {{-xml-schema-datatypes}} section 3.3.4) to ensure compatibility with the SWID specification where these names are used.
+
 {: #model-extension}
 ## Concise SWID Extensions
 
 The CoSWID data definition contains two features that are not included in the SWID data definition on which it is based. These features are:
 
-- The explicit definition of types for attributes that are typically stored in
-  the "any attribute" of an ISO-19770-2:2015 in XML representation. These are
+- The explicit definition of types for attributes that are typically represented in
```
Comment (ajnelson-nist) [inline draft-ietf-sacm-coswid.md:279]: This sentence needs a light grammar adjustment.

File: draft-ietf-sacm-coswid.md [hunk 2]
```diff
@@ -701,9 +670,9 @@ The following describes each child item of this group.
 
 - colloquial-version (index 45): A textual value for the software component's informal or colloquial version. Examples may include a year value, a major version number, or similar value that are used to identify a group of specific software component releases that are part of the same release/support cycle. This version can be the same through multiple releases of a software component, while the software-version specified in the concise-swid-tag group is much more specific and will change for each software component release. This version is intended to be used for string comparison only and is not intended to be used to determine if a specific value is earlier or later in a sequence.
 
-- description (index 46): A textual value that provides a detailed description of the software component. This value MAY be multiple sentences.
+- description (index 46): A textual value that provides a detailed description of the software component. This value MAY be multiple paragraphs seperated by CR LF characters as described by {{RFC5198}}.
```
Comment (ajnelson-nist) [inline draft-ietf-sacm-coswid.md:673]: Typo: separated.

File: draft-ietf-sacm-coswid.md [hunk 3]
```diff
@@ -1271,54 +1263,46 @@ are provided below. Assignments consist of an integer Index value, the Relations
...
 | 10          | supersedes               | See {{indexed-link-rel}}
 | 11          | supplemental             | See {{indexed-link-rel}}
-| 12-16384    | Unassigned               |
-| 16385-32767 | Unassigned               |
-| 32768-65535 | Reserved for Private Use |
+| 12-65535    | Unassigned               |
 {: #tbl-iana-link-rel-values title="CoSWID Link Relationship Inital Registrations"}
 
-Additional syntax requirements for registrations:
+Registrations MUST conform to the expert review guidlines defined in {{iana-review-guidelines}}.
```
Comment (ajnelson-nist) [inline draft-ietf-sacm-coswid.md:1269]: Typo: guidelines.

File: draft-ietf-sacm-coswid.md [hunk 4]
```diff
@@ -1393,6 +1377,68 @@ preferably with the specific value requested:
...
+
+Defining Specification: RFC-AAAA
+
+Deriving Software Identifiers:
+
+  A Software Identifier generated from a CoSWID tag is expressed as a concatenation of the form:
+
+    TAG_CREATOR_REGID "_" "_" UNIQUE_ID
+
+  Where TAG_CREATOR_REGID is the reg-id ietm value of the tag's entity item having the role value of 1 (corresponding to "tag creator"), and the UNIQUE_ID is the same tag's tag-id item. If the tag-id item's value is expressed as a 16 byte binary string, the the UNIQUE_ID MUST be represented using the UUID string representation defined in {{RFC4122}} including the "urn:uuid:" prefix.
```
Comment (ajnelson-nist) [inline draft-ietf-sacm-coswid.md:1438]: Typo: "the the".

File: draft-ietf-sacm-coswid.md [hunk 5]
```diff
@@ -1405,14 +1451,17 @@ security considerations regarding SWID/CoSWID tags focus on the application
...
-In this way, an authoritative SWID/CoSWID tag contains information about a software component provided by the maintainer of the software component, who is expected to be an expert in their own software. Thus, authoritative SWID/CoSWID tags can be trusted to represent authoritative information about the software component.  Having an authoritative SWID/CoSWID tag can be useful when the information in the
-tag needs to be trusted, such as when the tag is being used to convey
-reference integrity measurements for software components.  By contrast, the data contained in unsigned
+A tag is cnsidered "authoritative" if the SWID/CoSWID tag was created by the
+software provider. An authoritative SWID/CoSWID tag contains information about a software component provided by the maintainer of the software component, who is expected to be an expert in their own software. Thus, authoritative SWID/CoSWID tags can be trusted to represent authoritative information about the software component.  
+
+A signed SWID/CoSWID tag (see {{appendix-cose}}) whose signature has been validated can be relied upon to be
+unchanged since it was signed. By contrast, the data contained in unsigned o
```
Comment (ajnelson-nist) [inline draft-ietf-sacm-coswid.md:1458]: Typo? Dangling `o`.

File: draft-ietf-sacm-coswid.md [hunk 6]
```diff
@@ -1405,14 +1451,17 @@ security considerations regarding SWID/CoSWID tags focus on the application
 of SWID/CoSWID tags to address security challenges, and the possible
 disclosure of the results of those applications.
 
-A signed SWID/CoSWID tag whose signature has been validated can be relied upon to be
-unchanged since it was signed.  If the SWID/CoSWID tag was created by the
-software provider, is signed, and the software provider can be authenticated as the originator of the signature, then the tag can be considered authoritative.
-In this way, an authoritative SWID/CoSWID tag contains information about a software component provided by the maintainer of the software component, who is expected to be an expert in their own software. Thus, authoritative SWID/CoSWID tags can be trusted to represent authoritative information about the software component.  Having an authoritative SWID/CoSWID tag can be useful when the information in the
-tag needs to be trusted, such as when the tag is being used to convey
-reference integrity measurements for software components.  By contrast, the data contained in unsigned
+A tag is cnsidered "authoritative" if the SWID/CoSWID tag was created by the
```
Comment (ajnelson-nist) [inline draft-ietf-sacm-coswid.md:1454]: Typo: "cnsidered".

File: draft-ietf-sacm-coswid.md [hunk 7]
```diff
@@ -1461,10 +1510,14 @@ employ input sanitizing and loop detection on the tags they ingest.
 
 #  Acknowledgments
 
-TBD
+This document draws heavily on the concepts defined in the ISO/IEC 19770-2:2015 specification. The authors of this document are greatful for the prior work of the 19770-2 contributors.
```
Comment (ajnelson-nist) [inline draft-ietf-sacm-coswid.md:1513]: Typo: "greatful".

File: draft-ietf-sacm-coswid.md (hunk 1 above)
Comment (henkbirkholz) [reply draft-ietf-sacm-coswid.md:279]: I think you are correct. Currently, I have a hard time parsing that myself. I'll merge this for now and keep this issue open.

**Review by henkbirkholz — APPROVED** [review]
I agree with all of Alex's comments, which are not addressed by this current merge and have to be addressed after merging. There are some new rage definitions for values, e.g. -256 - 65536 (if I remember correctly) these can be include in the CDDL specification with controls.

Text improvement is significant and strongly accepted.

**Pull request merged**